charts/stable/strapi/values.yaml

```diff
@@ -16,7 +16,7 @@ env:
   DATABASE_PORT: 5432
   DATABASE_NAME: "{{ .Values.postgresql.postgresqlDatabase }}"
   DATABASE_USERNAME: "{{ .Values.postgresql.postgresqlUsername }}"
-  NODE_ENV: "production"
+  NODE_ENV: "development"
```

Comment:
?

Reply:
Run production fails, need jwt secret.

Reply:
It's should be addressed then :)

Reply:
the jwt token is done via the strapi gui, its not available as a ENV.

Reply:
I dont want development ENV in production/stable.